iphone/iphone/Titanium.entitlements

```diff
@@ -1,5 +1,24 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
-<dict/>
+<dict>
```

Comment:
These should be removed as we are excluding mac target in this PR.

Reply:
Yeah, this should probably be in the macOS support PR.